modules/nodes-pods-configmap-creating-from-literal-values.adoc

```diff
@@ -8,9 +8,9 @@
 
 You can supply literal values for a config map.
 
-.Procedure
+The `--from-literal` option takes a `key=value` syntax, which allows literal values to be supplied directly on the command line.
```

Comment:
I would move this sentence up with the "You can supply literal values for a config map." sentence, but thats up to you!